luapgsql.c

```diff
@@ -32,7 +32,15 @@
 #define htobe64(x) OSSwapHostToBigInt64(x)
 #elif __linux__
 #include <endian.h>
+#elif _WIN32
+#include <sys/param.h>
```

Comment:
This doesn't seem to be a standard windows header either :(

Googling around it seems that including winnt.h and then using `#if REG_DWORD == REG_DWORD_LITTLE_ENDIAN` is the correct thing?

Reply:
Okay, it still works. I could not include winnt.h directly. Instead windows.h includes windef.h and this includes winnt.h, finally.

In fact I am not that much into C programming. So thanks for your advices.